### PR TITLE
clean en.yml

### LINKED ISSUE #4997: Clean up en.yml
Created specifically for @Anjalizi

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

Problem: With the new dynamic stats page there are a few thing we are not using on https://github.com/publiclab/plots2/blob/master/config/locales/en.yml
 Lets clean up this

- [ ] 📝 **Update**: 
Remove this blocks of code
*  https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L250-L271
*  https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L276-L281
*  https://github.com/publiclab/plots2/blob/master/config/locales/en.yml#L283-L288

After changing this, attach the screenshot of `/stats` in the PR.  Just to make sure we didn't break anything on this page

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

Thanks!

### PR DESCRIPTION
**Pull request by Anjalizi**
Fixes #4997 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] PR is descriptively titled 📑
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

Below is the screenshot of the /stats page
stats

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by Anjalizi** [comment]
@gauravano @cesswairimu please review. thanks.

File: config/locales/en.yml [hunk 1]
```diff
@@ -247,45 +247,14 @@ en:
...
-      notes_and_wiki_edits_past_week: |
-        <a href='%{url1}'>%{note_count} notes</a>
-        and <a href='%{url2}'>%{wiki_count} wiki edits</a> posted in the past week.
-      people_joined_past_week: "<b>%{people_count} people</b> joined in the past week."
-      notes_and_wiki_edits_past_month: |
-        <a href='%{url1}'>%{note_count} notes</a>
-        and <a href='%{url2}'>%{wiki_count} wiki edits</a> posted in the past month.
-      people_joined_past_month: "<b>%{people_count} people</b> joined in the past
-        month."
+
```
Comment (gauravano) [inline config/locales/en.yml]:
```suggestion
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -247,45 +247,14 @@ en:
...
         posted by %{contributors_count} contributors since %{time} ago"
       total_contributors: "%{all_contributors_count} Total Contributors (Tallied weekly)"
       notes_per_week_past_year: "%{note_count} notes per week over the past year."
-      note_posting_52_weeks: 'Note posting for the past 52 weeks:'
-      wiki_pages_posted: "%{count} wiki pages have been posted, total"
-      wiki_page_posting_52_weeks: 'Wiki page posting for the past 52 weeks:'
-      comments_posted: "%{count} comments have been posted, total"
-      comments_posted_52_weeks: 'Comments posted for the past 52 weeks:'
-      wiki_page_edits: "%{count} wiki page edits have been made, total"
+
```
Comment (gauravano) [inline config/locales/en.yml]:
```suggestion
```

File: config/locales/en.yml [hunk 3]
```diff
@@ -247,45 +247,14 @@ en:
...
-      wiki_page_edits: "%{count} wiki page edits have been made, total"
+      
       edits_per_week: "%{count} edits per week over the past year."
-      questions_past_week: |
-                <a href='%{url1}'>%{quiz_count} questions asked </a>
-                and <a href='%{url2}'>%{ans_count} questions answered </a> in the past week.
-      questions_past_month: |
-                <a href='%{url1}'>%{quiz_count} questions asked </a>
-                and <a href='%{url2}'>%{ans_count} questions answered </a> in the past month.
+
```
Comment (gauravano) [inline config/locales/en.yml]:
```suggestion
```

**Comment by gauravano** [comment]
hey @Anjalizi, nice work, please also remove the empty lines too. Thanks!

**Comment by Anjalizi** [comment]
one out of three checks passed out of your suggested changes. @gauravano. how can it be rectified at my end?

**Comment by gauravano** [comment]
Actually, as you merged suggested commits one after another so build for the previous commit doesn't get started and got cancelled - https://travis-ci.org/publiclab/plots2/builds/504837453?utm_source=github_status. But, that's not a issue. We are all done here. Thanks!

**Comment by gauravano** [comment]
Merged :tada: :balloon: :100: :+1: 

@Anjalizi  thanks for the nice work. We would love to have more help from you. If you are interested in solving more issues then, check here - https://code.publiclab.org/#r=all for a new issue. In case, you are unable to find a issue, feel free to comment below. Thanks!

**Comment by Anjalizi** [comment]
thank you @gauravano, I shall find one from here.

**Pull request merged**